game/scripts/npc/heroes/queenofpain.txt

```diff
@@ -8,8 +8,7 @@
     // Abilities
     //-------------------------------------------------------------------------------------------------------------
     "Ability11"   "special_bonus_strength_16" // replaces special_bonus_strength_11
-
-    "Ability12"   "special_bonus_unique_queen_of_pain_4"
     "Ability13"   "special_bonus_attack_speed_60" // replaces special_bonus_attack_speed_30
+    "Ability16"   "special_bonus_unique_queen_of_pain_5" // replaces "special_bonus_unique_queen_of_pain_6"
```

Comment:
did you test the talent? QoP ult pushes people back already and combining it with fear might make the ability very buggy.

Reply:
1.4 knockback and than what is essentially a .6 second fear? Is that worth a talent slot if it works how I think it works/

Reply:
You can act (attack, cast spells, use items) during knockback afaik. So the fear basically makes the knockback better.

Reply:
I have tested it, works fine afaik. Was more of a proof of concept than anything, probably won't be my final numbers.

Reply:
the real question is why are we buffing a hero that is already really strong?

Reply:
is this like a QoP is good in dota right now therefore she must be good in here question or do you actually think she is good?